setup.cfg

```diff
@@ -19,15 +19,14 @@ classifiers =
     Topic :: Database
     Programming Language :: Python
     Programming Language :: Python :: 3
-    Programming Language :: Python :: 3.6
-    Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
     Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11
 
 [options]
 packages = find:
-python_requires = >=3.7,<3.11
+python_requires = >=3.7,<3.12
 install_requires =
     urllib3~=1.26,>=1.21.1
```

Comment:
FYI, this `urllib3` limitation was introduced many years ago when pip had limited capability to resolve version conflicts. I think we are good to remove the constraints (with some manual tests) since pip can resolve to choose the right version of urllib3.
https://github.com/treasure-data/pytd/issues/14

Reply:
Thanks @chezou! I created ticket to track this! Kindly continue with the review 🙇

Reply:
Thanks for your consideration. 

Considering the current pip capability, giving a strict upper version requires frequent release; otherwise, pytd triggers version conflicts too often. Maybe preventing major upgrades for pandas and numpy would be reasonable now.